openQA-client-test.spec

```diff
@@ -0,0 +1,25 @@
+%define         short_name openQA-client
+Name:           %{short_name}-test
+Version:        4.6
+Release:        0
+Summary:        Test package for %{short_name}
+License:        GPL-2.0-or-later
+BuildRequires:  %{short_name} == %{version}
+ExcludeArch:    i586
+
+%description
+.
+
+%prep
+# workaround to prevent post/install failing assuming this file for whatever
+# reason
+touch %{_sourcedir}/%{short_name}
+
+%build
+/usr/share/openqa/script/client --help
+
+%install
+# disable debug packages in package test to prevent error about missing files
```

Comment:
error**s**?

Reply:
this is copied from other files. And no, there is only a *single* error :)

Reply:
I copied this from `openQA-worker-test.spec`. I guess both words are valid. I don't know if it will print an error for every missing file, or print an error and then lists the files.
Is this really important?

Reply:
just leave it